app/tasks/maintenance/backfill_champs_stable_id_task.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Maintenance
+  class BackfillChampsStableIdTask < MaintenanceTasks::Task
+    def collection
+      Dossier.select(:id)
+    end
+
+    def process(dossier)
+      if Champ.exists?(dossier_id: dossier.id, stable_id: nil)
+        Migrations::BackfillStableIdJob
+          .set(wait: rand(0...48).to_i.hours)
```

Comment:
là tu batch heure par heure, c'est volontaire ?
Tu peux directement faire : 

```suggestion
          .set(wait: rand(0..48.hours))
```

Reply:
Tu serais plus fin ? Comme toutes les 10 ou 20 minutes ?